tests/components/treemap-tests.js

```diff
@@ -46,19 +46,30 @@ const TREEMAP_PROPS = {
   }
 };
 
+function size(node) {
+  return !node.children ? node.size : node.children.reduce((sum, child) => sum + size(child), node.size || 0);
+}
+
+function toString(children) {
+  const childStrings = children.sort((a, b) => size(b) - size(a)).map(child => {
+    return {title: child.title, childrenText: child.children && toString(child.children)};
+  });
+  return `${childStrings.map(child => child.title).join('')}${childStrings.map(child => child.childrenText || '').join('')}`;
+}
+
 // make sure that the components render at all
 testRenderWithProps(Treemap, TREEMAP_PROPS);
 
 test('Treemap: Basic rendering', t => {
   const $ = mount(<Treemap {...TREEMAP_PROPS}/>);
   t.equal($.find('.rv-treemap__leaf').length, 21, 'should find the right number of children');
-  const expectedText = 'EasingNeonateinterpolateISchedulableParallelPauseFunctionSequenceSequenceTransitionTransitionerTransitionEventSchedulerArrayInterpolatorColorInterpolatorDateInterpolatorInterpolatorMatrixInterpolatorNumberInterpolatorObjectInterpolatorPointInterpolatorRectangleInterpolator';
+  const expectedText = toString(TREEMAP_PROPS.data.children, 0);
```

Comment:
looks like this 0 is unused?

Reply:
also i'd maybe advise against having these functions generate test results? It makes it harder to update the tests?

Reply:
Yeah you're right about the 0, I'll just move those back to strings instead of the functions anyway.